cli/crates/federated-dev/src/dev/websockets.rs

```diff
@@ -170,20 +149,32 @@ async fn subscription_loop(
     sender.send(Message::Complete { id }).await.ok();
 }
 
-async fn accept_websocket(mut websocket: WebSocket) -> Option<(WebSocket, Option<serde_json::Value>)> {
+async fn accept_websocket(mut websocket: WebSocket, gateway: &GatewayWatcher) -> Option<(WebSocket, Session)> {
     while let Some(event) = websocket.recv_graphql().await {
         match event {
             Event::ConnectionInit { payload } => {
+                let Some(gateway) = gateway.borrow().clone() else {
+                    websocket
+                        .send(Message::close(4403, "Forbidden").try_into().unwrap())
```

Comment:
Internal server error (503) maybe?

Reply:
This is a websocket error code not an http error code - they're different.

(This being forbidden & 4403 was a mistake, but I've fixed that now)